apps/field.go

```diff
@@ -87,9 +87,12 @@ type Field struct {
 	// autocomplete.
 	SelectRefresh bool `json:"refresh,omitempty"`
 
-	// SelectStaticOptions is the list of options to display in a static select
-	// field.
-	SelectStaticOptions []SelectOption `json:"options,omitempty"`
+	// SelectOptions is the list of options to display in a static select field.
+	SelectOptions []SelectOption `json:"options,omitempty"`
```

Comment:
@larkox - why the renaming? 1/5 but I think it is easier to follow if we have `SelectStatisOptions` and `SelectLookupOptions` instead of `SelectOptions` and `SelectLookup`.